### PR TITLE
Modified docs to adhere to ibm style guide

### PR DESCRIPTION
**Pull request by jc-berger**
Fixes [#254](https://github.com/devfile/api/issues/254) Review Language in Authoring Stacks.

Since I am doing a review of all the authoring stacks docs, I'll be making several PRs. I don't want any single PR to be too large, so I'll make a new PR for every several docs that I modify to meet the IBM Style Guide's requirements.

### CODE REVIEW

**Review by rolfedh — APPROVED** [review]
One minor issue. Otherwise /lgtm

**Review by elsony — APPROVED** [review]
/lgtm

File: docs/modules/user-guide/partials/proc_adding-projects-to-a-devfile.adoc [hunk 1]
```diff
@@ -113,20 +111,19 @@ projects:
 ----
 ====
 
-. For each project, define the optional `sparseCheckoutDir` attribute to populate the project sparsely with selected directories.
+. For each project, define the optional `sparseCheckoutDir` attribute to populate the project with selected directories.
 +
 [NOTE]
 ====
-* Set to `/my-module/` to create only the root `my-module` directory (and its content).
+* Set your project to `/my-module/` to create only the root `my-module` directory along with its content.
```
Comment (rkratky) [inline docs/modules/user-guide/partials/proc_adding-projects-to-a-devfile.adoc:118]:
```suggestion
* Set the project to `/my-module/` to create only the root `my-module` directory along with its content.
```

File: docs/modules/user-guide/partials/proc_adding-projects-to-a-devfile.adoc [hunk 2]
```diff
@@ -113,20 +111,19 @@ projects:
...
 
-* Omit the leading slash (`my-module/`) to create all `my-module` directories that exist in the project. Including, for example, `/addons/my-module/`.
-+
-The trailing slash indicates that only directories with the given name (including their content) are created.
+* Omit the leading slash,`my-module/`, to create all `my-module` directories that exist in the project, including, `/addons/my-module/`.
 
-* Use wildcards to specify more than one directory name. For example, setting `module-*` checks out all directories of the given project that start with `module-`.
+** The trailing slash indicates that only directories and their content with the given name are created.
```
Comment (rkratky) [inline docs/modules/user-guide/partials/proc_adding-projects-to-a-devfile.adoc:122]:
```suggestion
** Add a trailing slash to create only directories with the given name (and their content).
```

File: docs/modules/user-guide/partials/proc_adding-a-name-to-a-devfile.adoc [hunk 3]
```diff
@@ -36,7 +37,7 @@ metadata:
 
 [NOTE]
 
-For workspaces created using a factory, defining `name` or `generateName` has the same effect. The defined value is used as the name prefix: `<name>YYYYY` or `<generateName>YYYYY`. When both `generateName` and `name` are defined, `generateName` takes precedence.
+If you create workspaces using a factory, defining `name` or `generateName` has the same effect. Use the defined value as the name prefix: `<name>YYYYY` or `<generateName>YYYYY`. If you define both `generateName` and `name`, then the `generateName` takes precedence.
```
Comment (rkratky) [inline docs/modules/user-guide/partials/proc_adding-a-name-to-a-devfile.adoc:40]: Getting rid of passive in favor of an 'if' clause is not an improvement. Also the second sentence would change the meaning.

```suggestion
Defining `name` or `generateName` has the same effect when creating workspaces using a factory. The value forms the name prefix: `<name>YYYYY` or `<generateName>YYYYY`. When defining both `generateName` and `name`, `generateName` takes precedence.
```

**Review by rkratky — APPROVED** [review]
@jc-berger, pre-approving, but have a look at the comment, please.

**Comment by jc-berger** [comment]
Due to the rebase issues I've had with this PR, I've made a new one. All the doc changes made in this PR can be found in #65: https://github.com/devfile/docs/pull/65 

Closing this PR since we'll work on and merge #65 instead.

**Pull request closed without merging**